src/main/java/com/google/sps/servlets/DeselectProjectServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import com.google.gson.Gson;
+import com.google.sps.data.User;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+import com.google.cloud.datastore.DatastoreException;
+		
+@WebServlet("/deselect")
+public class DeselectProjectServlet extends HttpServlet {
+
+	@Override
+	public void doGet(HttpServletRequest request, HttpServletResponse response) 
+			throws IOException {
+    		 // Allow CORS so frontend can access it			
+		 response.addHeader("Access-Control-Allow-Origin", "*");
+		 response.addHeader("Access-Control-Allow-Headers", 
+				"Origin, X-Requested-With, Content-Type, Accept, Authorization");
+		 response.addHeader("Access-Control-Allow-Credentials", "true");
+		 response.addHeader("Access-Control-Allow-Methods", "GET,POST,PUT,DELETE,OPTIONS,HEAD");
```

Comment:
Consider making a common utility method you can use across all your classes.

Reply:
This was not addressed. It's not good practice to duplicate code, it's much easier to have reusable methods or even one class that all your servlets inherit from. That way if in the future we need to make any changes, you only have to update in one place rather than in all your classes.

Reply:
I tried looking online to find a way to add response headers to every servlet and it requires implementing a doFilter which is associated with web.xml. Because of how close the deadline is and how many other base features we still need to implement, I didn't want to have the backend miss out on key features we need to get the application running trying to figure out doFilter. If we have time left before the deadline I will return this, but since the backend works as is I felt it would be better to have duplicate code for now and the application work than try to figure out doFilter (however long that would take) and no backend running.